src/Build/BuildCop/Infrastructure/EditorConfig/IEditorConfigParser.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.BuildCop.Infrastructure.EditorConfig
+{
+    public interface IEditorConfigParser
```

Comment:
```suggestion
    internal interface IEditorConfigParser
```

We'll expose only the bare minimum that's needed for authoring cusom analyzers. Rest will remain internal.
Whatever we'll expose, will need to be in `Microsoft.Build.Experimental.BuildCop` namespace - in order to be able to perform changes in the future.